scalameta/trees/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -364,6 +364,9 @@ object TreeSyntax {
         case t: Pat.Xml              =>
           val zipped = t.parts.zip(t.args).map{ case (part, arg) => s(part, "${", arg, "}") }
           m(SimplePattern, s(r(zipped), t.parts.last))
+        case Pat.Typed(lhs, rhs : Lit) =>
+          if (dialect.allowLiteralTypes) m(Pattern1, s(p(SimplePattern, lhs), kw(":"), " ", s(rhs)))
+          else throw new UnsupportedOperationException(s"$dialect doesn't support with types")
```

Comment:
I am not able to pass over this issue :(

<img width="1026" alt="captura de pantalla 2017-02-06 a las 21 30 17" src="https://cloud.githubusercontent.com/assets/10242025/22665093/c888d294-ecb3-11e6-88da-47bd5144c69d.png">